mayastor/src/bdev/nexus/nexus_label.rs

```diff
@@ -567,8 +642,8 @@ pub struct GptName {
 ///
 /// The struct should have a 440 byte code section here as well, this is
 /// omitted to make serialisation a bit easier.
-#[derive(Serialize, Deserialize)]
-struct Pmbr {
+#[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct Pmbr {
```

Comment:
Why do these things need to become pub?

Reply:
Pmbr needs to be pub as it is now part of the NexusLabel struct so that it can be read and checked for consistency with the GPT header. However all fields that don't *need* to be public at present have been made private.